src/cartesi-machine.lua

```diff
@@ -161,71 +158,42 @@ where options are:
 
     --rollup-rx-buffer=start:0x60000000,length:2<<20
     --rollup-tx-buffer=start:0x60200000,length:2<<20
-    --rollup-input-metadata=start:0x60400000,length:4096
-    --rollup-voucher-hashes=start:0x60600000,length:2<<20
-    --rollup-notice-hashes=start:0x60800000,length:2<<20
     --htif-yield-manual
     --htif-yield-automatic
 
   --rollup-advance-state=<key>:<value>[,<key>:<value>[,...]...]
     advances the state of the machine through a number of inputs in an epoch
 
     <key>:<value> is one of
-        epoch_index:<number>
         input:<filename-pattern>
-        input_metadata:<filename-pattern>
         input_index_begin:<number>
         input_index_end:<number>
-        voucher:<filename-pattern>
-        voucher_hashes: <filename>
-        notice:<filename-pattern>
-        notice_hashes: <filename>
-        report:<filename-pattern>
+        output:<filename-pattern>
+        output_hash:<filename-pattern>
         hashes
 
-        epoch_index
-        the index of the epoch (the value of %%e).
-
-        input (default: "epoch-%%e-input-%%i.bin")
-        the pattern that derives the name of the file read for input %%i
-        of epoch index %%e.
+        input (default: "input-%%i.bin")
+        the pattern that derives the name of the file read for input %%i.
 
         input_index_begin (default: 0)
         index of first input to advance (the first value of %%i).
 
         input_index_end (default: 0)
         one past index of last input to advance (one past last value of %%i).
 
-        input_metadata (default: "epoch-%%e-input-metadata-%%i.bin")
-        the pattern that derives the name of the file read for
-        input metadata %%i of epoch index %%e.
-
-        voucher (default: "epoch-%%e-input-%%i-voucher-%%o.bin")
-        the pattern that derives the name of the file written for voucher %%o
-        of input %%i of epoch %%e.
-
-        voucher_hashes (default: "epoch-%%e-input-%%i-voucher-hashes.bin")
-        the pattern that derives the name of the file written for the voucher
-        hashes of input %%i of epoch %%e.
+        output (default: "input-%%i-output-%%o.bin")
+        the pattern that derives the name of the file written for output %%o
+        of input %%i.
 
-        notice (default: "epoch-%%e-input-%%i-notice-%%o.bin")
-        the pattern that derives the name of the file written for notice %%o
-        of input %%i of epoch %%e.
-
-        notice_hashes (default: "epoch-%%e-input-%%i-notice-hashes.bin")
-        the pattern that derives the name of the file written for the notice
-        hashes of input %%i of epoch %%e.
-
-        report (default: "epoch-%%e-input-%%i-report-%%o.bin")
-        the pattern that derives the name of the file written for report %%o
-        of input %%i of epoch %%e.
+        output_hash (default: "input-%%i-output-hash.bin")
```

Comment:
This is the merkle root hash of the hashes of all outputs generated for that input? Maybe use the name "outputs-root-hash" instead of just "output-hash"?